front/app/modules/commercial/project_folders/citizen/components/ProjectFolderModerationRightsReceivedNotification/index.tsx

```diff
@@ -1,29 +1,18 @@
-import React, { ReactElement } from 'react';
-import {
-  TNotificationData,
-  IProjectFolderModerationRightsReceivedNotificationData,
-} from 'services/notifications';
+import React from 'react';
+import { IProjectFolderModerationRightsReceivedNotificationData } from 'services/notifications';
 import NotificationWrapper from 'containers/Navbar/components/NotificationMenu/components/NotificationWrapper';
 import messages from './messages';
 
 import { FormattedMessage } from 'utils/cl-intl';
 import T from 'components/T';
 
 interface Props {
-  notification:
-    | TNotificationData
-    | IProjectFolderModerationRightsReceivedNotificationData;
+  notification: IProjectFolderModerationRightsReceivedNotificationData;
 }
 
-export default function ProjectFolderModerationRightsReceivedNotification({
+const ProjectFolderModerationRightsReceivedNotification = ({
   notification,
-}: Props): ReactElement | null {
-  if (
-    notification.attributes.type !== 'project_folder_moderation_rights_received'
-  ) {
-    return null;
-  }
```

Comment:
Should this be removed?

Reply:
Good remark, still needed to add this. Yes, but we should have this check outside. If we are in this component, the type has to be that, so we should do it at the Outlet level. Just made a commit: https://github.com/CitizenLabDotCo/citizenlab/pull/304/commits/7f57056e88fd38fd7c11b2da36800b81950b2733

Still need to create the notificationType type, more specific than string.

Reply:
Also added the `TNotificationType`!